src/renderers/dom/shared/hooks/ReactDOMUnknownPropertyHook.js

```diff
@@ -67,6 +67,11 @@ if (__DEV__) {
     if (EventPluginRegistry.registrationNameModules.hasOwnProperty(name)) {
       return true;
     }
+    if (EventPluginRegistry.plugins.length === 0 && name.indexOf('on') === 0) {
```

Comment:
I think it would be nice to do `/^on[A-Z]/.test(name)` here -- that is, only counting properties with a capital letter after "on". Then `onions={true}` is caught. Since this is SSR only I guess it isn't a big deal.

lgtm otherwise, thanks!